docs/recipes.md

```diff
@@ -49,6 +49,7 @@ Guides, tips and strategies for getting the most out of your Foam workspace!
 
 ## Version control
 
+- Store your workspace in an auto-synced GitHub repo [[gistpad]]
```

Comment:
I think it makes sense to put it under "Version Control". 

@juanfrank77 any thoughts?

Reply:
Agreed. That's the right category to put it in.